contracts/tokens.js

```diff
@@ -105,6 +108,198 @@ actions.createSSC = async () => {
       }
     }
   }
+
+  /**
+   * Adds the table to store authorized contracts to do certain actions, i.e. transfer
+   */
+  tableExists = await api.db.tableExists('authorizations');
+  if (tableExists === false) {
+    await api.db.createTable('authorizations', ['account', 'contract', 'version', 'symbol', 'action', 'type']);
```

Comment:
the array parameter in the createTable method is used to create indexes, unless you really need all these indexes, I'd suggest to keep only the ones that are important for the code execution

Reply:
Ops. Alright, I'll change that